.github/workflows/codeql.yml

```
@@ -0,0 +1,23 @@
+  contents: read
+  security-events: write
+jobs:
+  analyze:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd
+      - uses: github/codeql-action/init@7aefa1c9aed02ae41531ec219164e64a0f087410
+        with:
+          languages: python
+      - uses: github/codeql-action/analyze@7aefa1c9aed02ae41531ec219164e64a0f087410
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)